### DISCUSSION TITLE
Segmentation fault (core dumped)

### DISCUSSION THREAD

**Discussion by LiuPengPeter** [idaholab/moose#16222] [Q&A General] [answered]
Hello guys, when I run my .i file, it shows:

Initializing equation system                                                               [  1.28 s]
Segmentation fault (core dumped)

What's the erroe?

**Comment by dschwen** [comment, marked as answer]
You need to build a debug version of your app with `make METHOD=dbg` and then run it in a debugger. We have instructions on our website (search for "debugging").

**Reply by LiuPengPeter** [reply]
Thank you! I solved the problem.

**Reply by uchiha-JN** [reply]
Hello guys,
I had the same segmentation fault(core dumped) when i tried to run an input file. I used the debugger tutorial in example 21. However when i hit my first break point in the debugging process, i get an error like this

Assertion `_dynamic_n < N' failed.
/home/mfm_student/miniconda3/envs/moose/libmesh/include/metaphysicl/dynamic_std_array_wrapper.h, line 69, compiled Jul 14 2021 at 11:38:59

Thread 1 "babbler-dbg" hit Breakpoint 1, 0x00007fffec90e000 in PMPI_Abort () from /home/mfm_student/miniconda3/envs/moose/lib/libmpi.so.12

It does not seem to be from a .h or c++ file within my babbler app. Can anyone kindly help out with this error?

**Reply by GiudGiud** [reply]
please create another discussions thread. This is unlikely to be the same segfault and this thread is 10 months old.

**Reply by uchiha-JN** [reply]
Alright. Thank you
